doc/expressionLanguage.md

```diff
@@ -0,0 +1,112 @@
+# Measurement Transformation Expression Language
+
+## Index
+
+* [Overview](#overview)
+* [Expression execution](#execution)
+* [Description](#description)
+  * [Types](#types)
+  * [Values](#values)
+  * [Allowed operations](#operations)
+* [Examples of expressions](#examples)
+
+## <a name="overview"/> Overview
+The IoTAgent Library provides an expression language for measurment transformation, that can be used to adapt the
+information coming from the South Bound APIs to the information reported to the Context Broker. Expressions in this
+language can be configured for provisioned attributes as explained in the Device Provisioning API section in the
+main README.md.
+
+## <a name="execution"/> Expression execution
+
+Whenever a new measurement arrives to the IoTAgent for a device with declared expressions, all of the expressions for
+the device will be checked for execution: for all the defined active attributes containing expressions, the IoTAgent
+will check which ones contain expressions whose variables are present in the received measurement. For all of those
+whose variables are covered, their expressions will be executed with the received values, and their values updated in
+the Context Broker.
+
+E.g.: if a device with the following provisioning information is provisioned in the IoTAgent:
+```
+{
+   "name":"location",
+   "type":"geo:point",
+   "expression": "${latitude}, ${longitude}"
+},
+{
+   "name":"fillingLevel",
+   "type":"Number",
+   "expression": "@{$level / 100}",
+   "cast": "Number"
+},
+```
```

Comment:
Yes, you are right. We changed that in the original documentation and the examples remained with the original format.